lib/ovirt_metrics/configurator.rb

```diff
@@ -0,0 +1,24 @@
+module OvirtMetrics
+  class Configurator
+    def initialize
+      self.suppress_warnings = false
+      self.connection_specification_name = 'ovirt_metrics' if ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
I believe this is inconsistent across old rails versions... We had to do something different in the ancestry gem.  Let me look that one up